library/ssl_12_cli.c

```diff
@@ -1,5 +1,6 @@
 /*
- *  SSLv3/TLSv1 client-side functions
+ *  Client-specific handshake reading/writing functions
+ *  for TLS versions <= 1.2.
  *
```

Comment:
In commit “Populate TLS <= 1.2 handshake source files”:

> This commit moves generic/client/server handshake handling code from ssl_tls.c, ssl_cli.c and ssl_srv.c to the newly created files ssl_12_gen.c, ssl_12_cli.c and ssl_12_srv.c.

Maybe but I can't follow the diff (with or without `--color-moved`). The `_12` files started out as copies of other files. Why is stuff being added to them?

Reply:
They started out as entirely unmodified copies for the sake of the git history.

Reply:
I understand that the files start as unmodified copies and why, but precisely for that reason I also expect to see only deletions in 22598875438b0534cf3ddf89ff57d094affb15b0 (resp. b769db8c6628ece4feb6fe5b8b1edffa2b178e5b) - before it each piece of code from `ssl_tls.c` exists in two places, so I expect the commit to remove exactly one version of each block, and I'm also surprised to see additions here.

I think it would ease reviewing if the third step in splitting files (after copying to two new files, then renaming one of them to the old name) did nothing else than remove blocks of code from each and the now-duplicated files, and left all the rest (adapting comments in the header, moving common stuff to `ssl_internal.h`, etc.) to later commits.

That way if would be very easy to check programmatically that absolutely no new code is introduced, and that code can only be moved or duplicated - and we can again check programmatically which lines were duplicated (that should be only the first dozen lines in each file: initial comment block with license, plus bunch of `#include` directives).

Right now 22598875438b0534cf3ddf89ff57d094affb15b0 does just a bit too much for me to be conveniently 100% sure it does nothing wrong, while I think that would be easier if all it did was remove blocks of code.